pkg/extensions/config/config.go

```diff
@@ -15,15 +15,15 @@ type ExtensionConfig struct {
 type SearchConfig struct {
 	// CVE search
 	CVE    *CVEConfig
-	Enable bool
+	Enable *bool
 }
 
 type CVEConfig struct {
-	UpdateInterval time.Duration // should be 2 hours or more, if not specified default be kept as 24 hours
+	UpdateInterval time.Duration // should be 2 hours or more, if not specified default be kept as 2 hours
```

Comment:
default should be 24 hours for cve updates, should not go faster. At least this comment needs to be reverted.